src/languages/ux_markup.coffee

```diff
@@ -2,7 +2,7 @@ module.exports = {
 
   name: "UX Markup"
   namespace: "ux"
-  fallback: ['html']
+  fallback: ['xml']
```

Comment:
This may change the behaviour for current users of `UX Markup`. Did this solve anything by changing it to `xml`?

Reply:
> I just figured this grammar was more closely related to XML than HTML.

This is true, however any users who are leveraging the `html` fallback will experience an unforeseen breaking change.

> It did not appear to have any effect in the build.

Unfortunately, our builds do not have great coverage 😝 . Something I hope to address with Unibeautify.